atomkraft/test/trace.py

```diff
@@ -55,4 +56,29 @@ def test_trace(trace: PathLike, reactor: PathLike, keypath: str):
             )
         )
     print(f"Executing {test_name} ...")
-    pytest.main(["-s", test_path])
+
+    report_dir = root / "reports" / test_name
+    report_dir.mkdir(parents=True)
+
+    logging_file = report_dir / "log.txt"
+
+    pytest_report_file = report_dir / "report.jsonl"
```

Comment:
Everything else (e.g. `stdout`, `stderr`, pytest report) is recorded in `pytest_report_file`.

Reply:
`logging` data is also captured here. But `pytest_report_file` is JSON Lines file. One has to process it with JSON processor to filter the outputs. `logging_file` stores `logging` data in simple text.